processor/groupbyattrsprocessor/processor.go

```diff
@@ -103,33 +103,152 @@ func (gap *groupByAttrsProcessor) processLogs(ctx context.Context, ld pdata.Logs
 
 	// Copy the grouped data into output
 	groupedLogs := pdata.NewLogs()
-	extractedGroups.MoveAndAppendTo(groupedLogs.ResourceLogs())
+	groupedResourceLogs.MoveAndAppendTo(groupedLogs.ResourceLogs())
 	stats.Record(ctx, mDistLogGroups.M(int64(groupedLogs.ResourceLogs().Len())))
 
 	return groupedLogs, nil
 }
 
+func (gap *groupByAttrsProcessor) processMetrics(ctx context.Context, md pdata.Metrics) (pdata.Metrics, error) {
+	rms := md.ResourceMetrics()
```

Comment:
This requires more knowledge about metrics than what I currently have, I trust the other reviewers took care of this already :-)